app/controllers/case_workers/admin/case_workers_controller.rb

```diff
@@ -5,7 +5,8 @@ class CaseWorkers::Admin::CaseWorkersController < CaseWorkers::Admin::Applicatio
 
   def index
     @case_workers = CaseWorker.includes(:location).joins(:user)
-    @case_workers = @case_workers.where("lower(users.first_name || ' ' || users.last_name) ILIKE :term", term: "%#{params[:search]}%") if params[:search].present?
+    query = t('sql.query.name')
```

Comment:
Not sure about using translations for SQL. I'd be tempted to put it in a HEREDOC method in the controller or a helper if necessary. Or if it is a complex query then a `query object`